src/components/shards/hero.tsx

```diff
@@ -10,31 +12,43 @@ export const Hero = async () => {
 
   return (
```

Comment:
brotha, if you're planning to use theming use the goddam variables

Reply:
kkkkk nice cooment